engine/include/cubos/engine/input/bindings.hpp

```diff
@@ -0,0 +1,115 @@
+/// @file
+/// @brief
```

Comment:
Don't forget to describe what is in the file here. Also, we need to have a convention for these commits, should we write:
```cpp
/// @file
/// Bla bla bla
```
or
```cpp
/// @file
/// @brief Bla bla bla